src/index.js

```diff
@@ -14,6 +14,7 @@ import * as Types from './constants/types';
 import * as Models from './models';
 
 import * as AnalysisApi from './api/AnalysisApi';
+import * as AppApi from './api/AppApi';
```

Comment:
Please add a `should expose AppApi` unit test for this in `/test/index.test.js`.